library/build.gradle.kts

```diff
@@ -78,3 +83,7 @@ publishing {
 signing {
     sign(publishing.publications.getByName("language-server-library"))
 }
+
+ktlint {
+    version = "1.2.1"
+}
```

Comment:
Is this block necessary?
Do we intend to hardcode future versions of ktlint like this?
How about extracting it with the rest of versions too?

Reply:
Nope, removed - the only block that remains required is the one ignoring BuildConfig stuff (they generate two-space indented code making ktlint upset)